### ISSUE TITLE
Internal server error on sentry_upload_dsym

### ISSUE THREAD

**Issue by olsh** [getsentry/sentry-fastlane-plugin#38]
This issue happens randomly.

Here is the log:
```
error: API request failed
caused by: sentry reported an error: request failure (http status: 500)
Object({"detail": String("Internal Error"), "errorId": String("e9a0d27c600847869c40805962ca6927")})
```

fastline 2.105.2
fastlane-plugin-sentry 1.5.0

**Comment by HazAT** [comment]
Thanks for reporting, we are aware of this issue and working on a fix.
If possible, please send your debug symbols (zipped) to daniel@sentry.io

**Comment by elviin** [comment]
I am also getting HTTP 500, no clue what is going wrong on the server. 

`ERROR [2019-08-16 08:30:15.48]: error: API request failed
ERROR [2019-08-16 08:30:15.48]: caused by: sentry reported an error: request failure (http status: 500)
ERROR [2019-08-16 08:30:15.48]: Object({"detail": String("Internal Error"), "errorId": String("5fa0b40e9063478ab3a.............")})`

....

ERROR [2019-08-16 08:30:15.48]: Error while calling Sentry CLI

Is there any way how to upload the symbols directly to the sentry website?

**Comment by HazAT** [comment]
@elviin Is is possible that you have the full error id?

**Comment by hartbit** [comment]
We are experiencing this very frequently on our hosted Sentry instance (about 80% of calls), causing us to have to retry our CI builds about 4-5 times before it succeeds:

```
Starting sentry-cli...
> Found 47 debug information files
> Prepared debug information files for upload
> Uploaded 42 missing debug information files
error: API request failed
```

How can we get more debug info to send you so we can see what is causing those failures?

**Comment by HazAT** [comment]
Export `SENTRY_LOG_LEVEL=debug` before you command and paste the output here.

**Comment by hartbit** [comment]
We set the log level and we only get one extra line printed:

```
DEBUG   2020-03-31 15:12:51.016703 +02:00 error: running update nagger
```

Does that help?

**Comment by hartbit** [comment]
@HazAT Any news? This is really causing us a lot of pain and we would be delighted for any kind of workaround. Thanks!

**Comment by philipphofmann** [comment]
This seems to be an issue in combination with sentry-cli. A lot has changed since the issue was reported. We recently released [sentry-cli 2.0.0](https://github.com/getsentry/sentry-cli/releases/tag/2.0.0). I'm going to close this. Feel free to open this issue again if the problem persists.
